src/components/BitcoinIntegrationPage/ReleaseTimeline/index.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import transitions from "@site/static/transitions.json";
+import AnimateSpawn from "@site/src/components/Common/AnimateSpawn";
+import { motion } from "framer-motion";
+import ChromiumGA from "@site/static/img/bitcoin-integration/chromiumGA.png";
+import ChromiumBeta from "@site/static/img/bitcoin-integration/chromiumBeta.png";
+
+function Index() {
+  const timelineTasks = [
+    "Bug Fixes",
+    "Code Audits",
+    "Developer Feedback",
+    "Performance Updates",
+    "Operational Experience",
+  ];
+  const betaReleaseInformation = [
+    { title: "Bitcoin integration", body: "Bitcoin Testnet" },
+    {
+      title: "Threshold ECDSA",
+      body: "ECDSA Testkey, deployed on one Production ECDSA key, deployed on medium-replication-factor subnet",
+    },
+    {
+      title: "Example code & doc",
+      body: "Example dApps & web-based documentation",
+    },
+    { title: "On-chain-tradable asset", body: "n.a." },
+    {
+      title: "Purpose",
+      body: "Development, Bitcoin testing and threshold ECDSA canisters",
+    },
+    { title: "Status", body: "Experimental" },
+  ];
+  const gaReleaseInformation = [
+    { title: "Bitcoin integration", body: "Bitcoin Mainnet" },
```

Comment:
```suggestion
    { title: "Bitcoin integration", body: "Bitcoin Mainnet + Testnet" },
```

We'll be supporting both mainnet and testnet with the GA release.